src/Classes/ServiceAPI/MyRadio_Track.php

```diff
@@ -619,26 +619,37 @@ public static function cacheAndIdentifyUploadedTrack($tmp_path)
 
         $getID3 = new \getID3();
         $fileInfo = $getID3->analyze(Config::$audio_upload_tmp_dir.'/'.$filename);
+        $getID3_lib = new \getID3_lib();
+        $getID3_lib->CopyTagsToComments($fileInfo);
+        
 
         // File quality checks
         if ($fileInfo['audio']['bitrate'] < 192000) {
-            return ['status' => 'FAIL', 'error' => 'Bitrate is below 192kbps.', 'fileid' => $filename, 'bitrate' => $fileInfo['audio']['bitrate']];
-        }
-        if (strpos($fileInfo['audio']['channelmode'], 'stereo') === false) {
-            return ['status' => 'FAIL', 'error' => 'Item is not stereo.', 'fileid' => $filename, 'channelmode' => $fileInfo['audio']['channelmode']];
-        }
-
-        $analysis = self::identifyUploadedTrack(Config::$audio_upload_tmp_dir.'/'.$filename);
-        if (isset($analysis['status'])) {
+            return ['status' => 'FAIL', 'message' => 'Bitrate is below 192kbps', 'fileid' => $filename, 'bitrate' => $fileInfo['audio']['bitrate']];
+        } else if (strpos($fileInfo['audio']['channelmode'], 'stereo') === false) {
+            return ['status' => 'FAIL', 'message' => 'Item is not stereo', 'fileid' => $filename, 'channelmode' => $fileInfo['audio']['channelmode']];
+        } else {
+            $analysis['status'] = 'INFO';
+            $analysis['message'] = 'Currently editing track information for';
+            $analysis['submittable'] = True;
             $analysis['fileid'] = $filename;
-
+            $analysis['analysis']['title'] = $fileInfo['comments_html']['title'];
+            $analysis['analysis']['artist'] = $fileInfo['comments_html']['artist'];
+            $analysis['analysis']['album'] = $fileInfo['comments_html']['album'];
+            
+           
+            //Remove total tracks in album from the track_number tag.
+            $trackNo = explode("/", $fileInfo['comments_html']['track_number'][0], 2)[0];
+            $analysis['analysis']['position'] = (string)$trackNo;
+
+            $trackName = implode("", $fileInfo['comments_html']['title']);
+           
+            if (stripos($trackName, 'explicit') == true) {
+                $analysis['analysis']['explicit'] = true;
```

Comment:
I mean merge it all into one line. No need for `if (something) { foo = true; } else { foo = false;}` just do `foo = something;`